.github/workflows/upgrade-ci.yml

```
@@ -0,0 +1,160 @@
+      - name: Use mtaVersion
+        run: echo "Konveyor version is $mtaVersion"
+
+      - name: Check pod status
+        run: kubectl get pods -n konveyor-tackle
+
+      - name: Wait for Ingress
+        shell: bash
+        run: |
+          bash -c 'external_ip="";
+          echo $external_ip;
+          while [[ -z $external_ip ]] 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)